src/modules/Blend/Module.js

```diff
@@ -1,7 +1,9 @@
 module.exports = function Dynamic(options, UI, util) {
 
-    options.func = options.func || "function(r1, g1, b1, a1, r2, g2, b2, a2) { return [ r1, g2, b2, a2 ] }";
-    options.offset = options.offset || -2;
+    var defaults = require('./../../util/getDefaults.js')(require('./info.json'));
+
+    options.func = options.func || defaults.func;
```

Comment:
Isn't this supposed to be 
`options.blend` instead of `options.func`? Also the tests are failing

Reply:
Yes, just made the changes. Thanks!